include/st20_api.h

```diff
@@ -455,15 +455,15 @@ struct st20_rfc4175_rtp_hdr {
   uint16_t row_number;
   /** Offset of the first pixel of the payload data within the scan line */
   uint16_t row_offset;
-} __attribute__((__packed__));
+});
 
 /**
  * A structure describing a st2110-22(video) rfc9134 rtp header, size: 16
  */
-struct st22_rfc9134_rtp_hdr {
+#ifdef MTL_LITTLE_ENDIAN
```

Comment:
is there any reason why move LITTLE_EDIAN out of the structure define?

Reply:
due to this - https://stackoverflow.com/questions/64452244/c-c-how-should-preprocessor-directive-work-on-macros-argument-list

Reply:
ok，thanks